autopilot_test.go

```diff
@@ -155,4 +49,25 @@ var _ = Describe("Command Syntax", func() {
 		args := cliConn.CliCommandArgsForCall(0)
 		Ω(args).Should(Equal([]string{"push", "-h"}))
 	})
+
+	It("can push an app that already exists", func() {
+		cliConn.CliCommandReturns([]string{"myapp and-other-stuff"}, nil)
+		output := CaptureOutput(func() {
+			autopilotPlugin.Run(cliConn, []string{"push-zdd", "myapp"})
+		})
+
+		Expect(len(ActionList)).To(Equal(3))
```

Comment:
rather than get the length of this ActionList array, which requires us to make it exported, you might consider just leveraging the cli plugin fakes.

`CliCommandCallCount ` will get you the call count
https://github.com/cloudfoundry/cli/blob/master/plugin/fakes/fake_cli_connection.go#L267

`CliCommandArgsForCall` will get you the actual argument sets called.
https://github.com/cloudfoundry/cli/blob/master/plugin/fakes/fake_cli_connection.go#L273